### PR TITLE
Add sompy

### PR DESCRIPTION
**Pull request by apcamargo**

### CODE REVIEW

**Comment by apcamargo** [comment]
Pinging @conda-forge/staged-recipes

**Comment by npavlovikj** [comment]
@apcamargo , I may be wrong, but I don't think the recipe was actually built. The CircleCI logs show "Invalid recipe dir: /home/conda/conda-recipes/sompy - skipping. Computed that there are 0 distributions to build from 1 recipes. Nothing to do."

**Comment by apcamargo** [comment]
You're right, @npavlovikj . @conda-forge/staged-recipes , what may be causing `invalid recipe dir`?

File: recipes/sompy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - scikit-learn >= 0.16
```
Comment (isuruf) [inline recipes/sompy/meta.yaml]: There should be no space between `>=` and `0.16`

**Comment by apcamargo** [comment]
Waiting for a fix:
https://github.com/conda/conda-build/issues/3287

**Comment by npavlovikj** [comment]
@apcamargo , if you are adding a LICENSE from the recipe dir, then you need to have "license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'" instead.
You can pretty much use the same recipe from bioconda here, https://github.com/bioconda/bioconda-recipes/blob/c05df4cd3ce7bd0b90a4ae01a96606bec0d12c61/recipes/sompy/meta.yaml, and just update the dependencies and the versions.

**Comment by npavlovikj** [comment]
@apcamargo , can you please restart the build (I don't have permissions to do that). @isuruf fixed the previous issue with the build system.

**Comment by apcamargo** [comment]
Thanks again, @npavlovikj .
The Windows build is still failing. I think it's another problem in the build system (though I'm not sure).

File: recipes/sompy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 846626a39e27cca171a81f4388cb30a32cdcd3bedf170c91227cf5a958aaabec
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```
Comment (scopatz) [inline recipes/sompy/meta.yaml]: probably should do `skip: True  # [py3k]` rather than the `python <3.0` below.

**Comment by isuruf** [comment]
Can this be `noarch: python` ?

**Comment by apcamargo** [comment]
@isuruf Can `noarch: python` and `skip: True # [py3k]` be used at the same time?

**Pull request merged**